tests/test_signal_timeout_block.py

```diff
@@ -192,10 +194,15 @@ def test_multiple_intervals(self):
 
     def test_persistence(self):
         """Persisted timeout jobs are notified accordingly"""
-        block = SignalTimeout()
+        class TestSignalTimeout(SignalTimeout):
```

Comment:
Oh wow, yeah you're right. When I was doing this with the `copy` version I was using events in the bloc to test scenarios. This ended up being pointless.